### DISCUSSION TITLE
`statsforecast` is now available on `conda-forge` channel to install with `conda`

### DISCUSSION THREAD

**Discussion by sugatoray** [Nixtla/statsforecast#55] [Show and tell]
🥇  `statsforecast` is now available on `conda-forge` channel to install with `conda`. Install the library with:

```sh
conda install -c conda-forge statsforecast
```

- https://anaconda.org/conda-forge/statsforecast

![image](https://user-images.githubusercontent.com/10201242/157090873-a047e032-7dd1-4286-bfc4-22e62c036b16.png)

**Comment by sugatoray** [comment]
cc: @FedericoGarza @mergenthaler

**Reply by AzulGarza** [reply]
Amazing, @sugatoray! Thank you very much :) It was really helpful

**Reply by sugatoray** [reply]
@FedericoGarza The conda-forge recipe is also setup such that on each PyPI release it will also self-update the conda-forge recipe (minimal human intervention).

**Reply by AzulGarza** [reply]
Cool! :)

**Reply by AzulGarza** [reply]
Hi @sugatoray! We've released a couple of versions but I think the conda-forge recipe has not been updated automatically. Do you have an idea of what is going on? Thank you!

**Reply by sugatoray** [reply]
Hi @FedericoGarza,

This is quite strange. I am not sure why it was not updated. Thank you, for pointing it out. I will send a PR to update the version in the conda-forge feedstock.

- https://github.com/conda-forge/statsforecast-feedstock/

**Comment by sugatoray** [comment]
I also posted this on LinkedIn. Sharing here for greater reach.

https://www.linkedin.com/posts/sugatoray_python-timeseries-conda-activity-6906674169338343424--4AG

![install_statsforecast](https://user-images.githubusercontent.com/10201242/157101051-a2be49cd-575d-4de8-8fd4-b06e6967999d.png)
